bin/wasm-node/rust/src/lib.rs

```diff
@@ -480,9 +480,9 @@ async fn handle_rpc(rpc: &str, client: &mut Client) -> (String, Option<String>)
                     ))
                     .to_json_response(request_id)
                 }
-                Err(error) => json_rpc::parse::build_error_response(
+                Err(_) => json_rpc::parse::build_error_response(
                     request_id,
-                    json_rpc::parse::ErrorResponse::ServerError(-32000, &error.to_string()),
+                    json_rpc::parse::ErrorResponse::ServerError(-32000, "announce_transaction failed"),
```

Comment:
```suggestion
                    json_rpc::parse::ErrorResponse::ServerError(-32000, "Failed to announce transaction"),
```

This is an error that users will see. There's no point in telling them `<internal detail> has failed`.